app/Models/User.php

```diff
@@ -138,6 +139,30 @@ public function studentSubscriptions(): MorphToMany
             ->withTimestamps();
     }
 
```

Comment:
Not used at this point, but let's also add the `careTeams(): HasMany` relationship.